db.js

```diff
@@ -12,7 +12,6 @@ const { indexesPath } = require('./defaults')
 const Log = require('./log')
 const BaseIndex = require('./indexes/base')
 const Private = require('./indexes/private')
-const Migrate = require('./migrate')
```

Comment:
The migrate plugin is no longer embedder inside db plugin. Instead, "ssb-db2" is db + migrate, see `index.js`.